frontend/src/components/room/QuestionModal.tsx

```diff
@@ -49,7 +51,7 @@ const QuestionModal = () => {
   const getQuestion = async () => {
     const res = await axios
       .get(URL_QUESTION_GET_RANDOM, {
-        headers: { difficulty: difficulty },
+        headers: { difficulty },
```

Comment:
If this works without the `difficulty` label then should be no problem

Reply:
Can change to `headers: { difficultyLevel: difficulty }`

Remember to change the corresponding header in backend (question-service/QuestionCtrl.ts line 73) to:
`const difficulty =  req.headers.difficultyLevel as string;`
<img width="480" alt="image" src="https://user-images.githubusercontent.com/77255760/195965010-a1ef7272-2975-474f-9ac6-d0bce55e2f81.png">